backend/bittan/bittan/services/swish/__init__.py

```diff
@@ -0,0 +1,163 @@
+from django.utils.log import logging
+from django.apps import apps
+import requests
+import json
+
+from ...models.swish_payment_request import SwishPaymentRequestModel, PaymentErrorCode as SwishApiErrorCode, PaymentStatus as SwishApiPaymentStatus
+from .swish_payment_request import SwishPaymentRequest
+
+from uuid import uuid4
+
+
+class SwishPaymentRequestResponse: 
+	""" This is the response that the official swish api sends after we have sent a request """ 
+	status: SwishApiPaymentStatus
+	error_code: SwishApiErrorCode 
+	id: str 
+
+	def __init__(self, response):
+		self.id = response['id']
+		self.status = SwishApiPaymentStatus.from_swish_api_status(response['status'])
+		self.error_code = SwishApiErrorCode.from_swish_reponse_code(response['errorCode'])
+
+class Swish:
+	def __init__(self, swish_url, payee_alias, callback_url, cert_file_paths, callback_function):
+		self.swish_url = swish_url
+		self.payee_alias = payee_alias
+		self.callback_url = callback_url
+		self.payeee_reference_id = ""
```

Comment:
double-check should this be an empty string? Also spell as payee_reference_id

Reply:
f03c4885255d42315474ae72a3122a0c897980c2